core/client/fs/src/test/java/alluxio/client/file/BaseFileSystemTest.java

```diff
@@ -86,8 +89,7 @@ public void before() {
   /**
    * Releases the client after a test ran.
    */
-  @After
-  public void after() {
+  public void verifyMasterClient() {
```

Comment:
Can we update the method name to give an indication of what is being verified? We aren't actually verifying anything about the master client itself, we're just verifying that the FileSystemContext has acquired and released it.

Reply:
how's `verifyFilesystemContextAcquiredAndReleased`